picasso/src/main/java/com/squareup/picasso3/Initializer.java

```diff
@@ -0,0 +1,4 @@
+package com.squareup.picasso3;
+
+public @interface Initializer {
```

Comment:
Add `@Retention(SOURCE)` and `@RestrictTo(LIBRARY)`

Reply:
done